index.html

```diff
@@ -1448,17 +1456,25 @@ <h2>The <dfn>NFCRecordType</dfn> enum</h2>
   <pre class="idl">
     typedef (DOMString or ArrayBuffer or NFCMessage) NFCMessageSource;
 
+    [Constructor(), SecureContext, Exposed=Window]
+    interface NFCWriter {
+      Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
+    };
+
     [SecureContext]
     interface NFC {
-      Promise&lt;void&gt; push(NFCMessageSource message, optional NFCPushOptions options);
       Promise&lt;void&gt; watch(MessageCallback callback, optional NFCWatchOptions options);
     };
 
     callback MessageCallback = void (NFCMessage message);
   </pre>
+  <p>
+    The <dfn>NFCWriter</dfn> is an object used for writing data to NFC devices
```

Comment:
Nit: Instances of NFCWriter are used to write data to NFC devices (this is not a singleton class)